pkg/utils/utils.go

```diff
@@ -165,6 +165,10 @@ func renderDisplayableList(items []Displayable, config RenderListConfig) (string
 
 // RenderTable takes an array of string arrays and returns a table containing the values
 func RenderTable(stringArrays [][]string) (string, error) {
+	// if empty array is given then getPadWidths will panic hence return an empty string
```

Comment:
I'd lose this comment too because getPadWidths panicking isn't really the reason we're returning an empty string: it's more that we know we don't need to render anything and I think the code is self-explanatory in that case